tests/basic_integration/fns.sh

```
@@ -0,0 +1,138 @@
+
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)